express-app/app.models.js

```diff
@@ -61,3 +62,29 @@ exports.fetchComments = (article_id) => {
     return rows;
   });
 };
+
+exports.postNewComment = ({ article_id, author, body }) => {
+  if (!author || !body) {
+    return Promise.reject({ status: 400, msg: 'Bad Request' });
+  }
+  if (body.length < 1) {
+    return Promise.reject({ status: 400, msg: 'Bad Request' });
+  }
+  const query = format(
+    'INSERT INTO comments (article_id, author, body) VALUES (%L, %L, %L) RETURNING *;',
+    article_id,
+    author,
+    body
+  );
+  return db.query(query).then(({ rows }) => {
+    const newComment = rows[0];
+    return {
```

Comment:
You can just return the comment that is retuned from the successful insertion. Votes will be 0 - look at the seed function to confirm this.